includes/Core/Admin/Plugin_Action_Links.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Admin\Plugin_Action_Links
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Admin;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Permissions\Permissions;
+
+/**
+ * Class for managing plugin action links.
+ *
+ * @since 1.41.0
```

Comment:
Since tags should always be added using the literal `n.e.x.t` placeholder which is then updated as part of the preparation for a release so you don't need to think about what release it might go in.